Features.md

```diff
@@ -0,0 +1,24 @@
+## JGit
+### Feature 1
+  #### Story A - Add Pull Command Line Command
+   - Task 1: Add a pull command to the CLI, callable with "jgit pull"
```

Comment:
Is theer a requirement for command line input, or is that an existing feature you are extending.

Reply:
That is an existing, but incomplete feature that I am adding onto.